MATLAB/environmental_models/env_atmospheric_drag.m

```diff
@@ -0,0 +1,89 @@
+function F_envdrag = env_atmospheric_drag(time,r,v)
+%input: mission time in sec
+% r position in ECI frame in m
+% v velocity in ECI frame in m/s
+% Omega longitude/right ascension of the ascending node in rad
+
+% outputs: Fenv_drag in N in direction of velocity % check dis ECI frame? 
+% assumes a simple, fully static exponentially decaying model (Vallado)
+
+global const
+%time conversions
+UTC_leap = utl_time2datetime(time,const.INITGPS_WN);  %converts mission time in secs to datetime in UTCleapseconds
+%UTC_leap = utl_time2datetime(time,3); test
+offset = 2*86400; %offset of UTCLeapSeconds to UTC time
+del = datetime(0,0,0,0,0,offset);
+del.TimeZone = 'UTCLeapSeconds';
+UTC = UTC_leap - offset;
+
+% r: satellite position in ECI
+%convert ECI coordinates to latitude, longitude, altitude (LLA) geodetic coordinates based on the Universal Coordinated Time (UTC) you specify
+lla = eci2lla(r, [UTC.Year UTC.Month UTC.Day UTC.Hour UTC.Minute UTC.Second]); 
```

Comment:
you have to transpose r so its a row vector, like `r'` to use `eci2lla`.

Reply:
updated